### ISSUE TITLE
Using 'j' expands a heading

### ISSUE THREAD

**Issue by g3g3n3** [aquirdTurtle/Collapsible_Headings#28]
As you probably know, Jupyter Lab allows you to vertically navigate cells with <j> and <k>, just like vim.

Is there a reason that the down arrow allows the user to select the heading below the current one without opening it, yet using <j> (like in vim) not only selects it, but expands it as well? 

It would be very nice if the <j> shortcut only selected a heading, and (if desired) we could separately open the heading  with the <t> shortcut (that you have built in).

Basically, I'd like to have the <j> key work the same as the down arrow (i.e. without opening the heading if it is collapsed)

**Comment by g3g3n3** [comment]
Could you change it so that that <j> doesn't open headings?
Also, perhaps <H> could close the heading and <L> could open it?
This would also make it more "vim-like".
Cheers.

**Comment by g3g3n3** [comment]
I just looked at the .json file an it appears that you don't have a <j> hotkey. So, this might be an issue with the default hotkey in Jupyter, or perhaps with the vim/jupyter plugin I have. Would be nice if somehow moving down with <j> didn't expand everything. If you are too busy, could you recommend a similar project to yours that is active? Thanks.

**Comment by g3g3n3** [comment]
Perhaps just adding <j> to the json file of your extension will override the default behavior.

**Comment by aquirdTurtle** [comment]
Hi @g3g3n3. I'm working on this now, and you're right it should be an easy fix. But for some reason some of the text you are writing here isn't showing up for me. I see you wrote " Also, perhaps could close the heading and could open it?", where presumably there were other specific keys you were suggesting to close and open the headings. What were those keys?

**Comment by aquirdTurtle** [comment]
Regardless, the "j"/"k" issues should be fixed in version 2.3.0. Let me know if it's not working for you or if there are any other issues.
